applocation.yml

```diff
@@ -0,0 +1,43 @@
+name: Erc20-analytics
+steps:
+  - type: trigger
+    instance:
+      src: https://github.com/mesg-foundation/service-ethereum-erc20
+      env:
+        - PROVIDER_ENDPOINT=$(env:PROVIDER_ENDPOINT)
+        - CONTRACT_ADDRESS=
+    eventKey: transfer
+  - type: task
+    key: transfer-data
+    instance:
+      src: https://github.com/mesg-foundation/service-js-function
+    taskKey: execute
+    inputs:
+      code: >
+        module.exports=(transferData) => ({
+          blockNumber: transferData.blockNumber,
+          transactionHash: transferData.transactionHash,
+          from: transferData.from,
+          to: transferData.to,
+          value: parseFloat(transferData.value),
+          contractAddress: transferData.contractAddress
+        })
+      inputs:
+        blockNumber: { key: blockNumber }
+        contractAddress: { key: contractAddress }
+        from: { key: from }
+        to: { key: to }
+        value: { key: value }
```

Comment:
I think this whole task is unnecessary. You can directly access the data from the trigger in the `influxdb#write` tasks

Reply:
the task `js-function` is using for convert value from `string` to `float`. The value data can save value as a string at `influxdb`?

Reply:
could you try to save it as string, maybe influxdb is automatically doing the conversion. In that case, we could remove this task, otherwsie, it's good like that.

Reply:
I test to save it as a string, the influxdb isn't doing the conversion so the Grafana isn't display any data.